### DISCUSSION TITLE
Add ControlD Free DNS Resolvers to Presets

### DISCUSSION THREAD

**Discussion by WingofaGriffin** [kkebo/DNSecure#93] [Ideas]
[ControlD](https://controld.com/free-dns) has a list of free DNS servers that they provide, that have been considered to be some of the best in terms of privacy and filtering: https://www.privacyguides.org/en/dns/

In my opinion, these are the best ad filtering DNS servers (I personally use the OISD one) as well as other filters with no logging and a solid Anycast network.

I'd suggest adding a few of these as preconfigured options. Happy to PR the code myself, but wanted to start a discussion :)

**Comment by kkebo** [comment]
Thanks for your suggestion.

Honestly, I am reluctant to make the preset list of servers long. That's because I originally thought of the presets as just examples to help people understand how to enter their settings.

Instead of having a long list by default, is there any direction to make it as easy as possible for people to enter their settings?
